### ISSUE TITLE
codingStandardsIgnoreStart / End

### ISSUE THREAD

**Issue by becoded** [CSSLint/csslint#248]
In the codesniffer of PHP you are able to say: 

``` php
/* @codingStandardsIgnoreStart */
$test = ‘something not conform the standards’;
/* @codingStandardsIgnoreEnd */
```

In the documentation of the PHP codesniffer it is described like this:

> Some parts of your code may be unable to conform to your coding standard. For example, you might have to break your standard to integrate with an external library or web service. To stop PHP_CodeSniffer generating errors for this code, you can wrap it in special comments. PHP_CodeSniffer will then hide all errors and warnings that are generated for these lines of code. 

So it just means that everything between those 2 tags is not tested.

This way, you are able to allow something that you really, really, really want but don’t want to get a warning/error about it but you know that it is something "bad"
.
Our developers are not allowed to commit any css that doesn’t follow some rules. 

We also want to enable the rule _known-properties_ but we can’t do it for the moment because of a horrible hack they are using for IE.
Eg: 
behavior: url(/rd.htc);   /_http://msdn.microsoft.com/en-us/library/ms530723%28v=vs.85%29.aspx_/

So if something exists like 

``` css
.rdBox {
    border: 1px solid #000;
    /* @codingStandardsIgnoreStart */
    behavior: url(/PIE.htc);   
    /* @codingStandardsIgnoreEnd */
}
```

then we can enable the rule without any problems. Every thing will be checked except for the thing you know that are bad but you really need.
